### ISSUE TITLE
ERROR in csp/slides/csnet-equivalent.en.xhtml

### ISSUE THREAD

**Issue by abchugh** [slatex/sTeX-React#30]
A content ERROR was logged by a user at the following url:

https://courses.voll-ki.fau.de/browser/%3AsTeX%2Fdocument%3Farchive%3DMiKoMH%2FAI%26filepath%3Dcourse%2Fnotes%2Fnotes.xhtml?inDocPath=-pg601z

## The issue as described by the user:
```
node and edge labels are not visible in constraint network graphs
```

## The text highlighted while reporting this issue:
```
γ
```

### The selected text was in the following section hierarchy:

**_INNERMOST SECTION FIRST_**

1. GitLab: https://gl.mathhub.info/MiKoMH/AI/-/blob/main/source/csp/slides/csnet-equivalent.en.tex<br/>FetchURL: /:sTeX/fulldocument?archive=MiKoMH/AI&filepath=csp/slides/csnet-equivalent.en.xhtml

2. GitLab: https://gl.mathhub.info/MiKoMH/AI/-/blob/main/source/csp/sec/csprop-inference.en.tex<br/>FetchURL: /:sTeX/fulldocument?archive=MiKoMH/AI&filepath=csp/sec/csprop-inference.en.xhtml

3. GitLab: https://gl.mathhub.info/MiKoMH/AI/-/blob/main/source/csp/sec/cspropagation.en.tex<br/>FetchURL: /:sTeX/fulldocument?archive=MiKoMH/AI&filepath=csp/sec/cspropagation.en.xhtml

4. GitLab: https://gl.mathhub.info/MiKoMH/AI/-/blob/main/source/course/notes/notes.tex<br/>FetchURL: :sTeX/fulldocument?archive=MiKoMH/AI&filepath=course/notes/notes.xhtml

**Comment by Jazzpirate** [comment]
right, this is a known issue. I think I need to do hacky things with respect to paragraphs in tikz/pgf/svg on the rustex side (the labels are there, but they're misplaced to the right of the circles)

**Comment by abchugh** [comment]
Seems fixed now:
https://stexmmt.mathhub.info/:sTeX/fulldocument?archive=MiKoMH/AI&filepath=csp/slides/csnet-equivalent.en.xhtml
